.github/workflows/frontend.yml

```diff
@@ -124,10 +124,19 @@ jobs:
           github-token: ${{ steps.token.outputs.token }}
 
       - name: tsc
+        id: tsc
         if: steps.dependencies.outcome == 'success' && needs.files-changed.outputs.frontend == 'true'
         run: |
           set -o pipefail
-          yarn tsc -p config/tsconfig.build.json --diagnostics | tee /tmp/typescript-monitor.log && yarn run ts-node .github/workflows/scripts/monitor-typescript.ts
+          yarn tsc -p config/tsconfig.build.json --diagnostics --generateTrace /tmp/trace | tee /tmp/typescript-monitor.log
```

Comment:
What are TSC profiles?

Reply:
`--generateTrace` emits a json file in the [trace event format](https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview). This can then be used to generate a flame chart that can help us identify where the time was spent. You can see an example that was generated [here](https://sentry.io/organizations/sentry-test/profiling/?project=6326775).